### ISSUE TITLE
Error when trying to execute a sample test - MLNET-CLI

### ISSUE THREAD

**Issue by julianobd** [dotnet/machinelearning#4213]
### System information

SDK do .NET Core (refletindo qualquer global.json):
 Version:   2.1.508
 Commit:    9ba8583e91

Ambiente de tempo de execução:
 OS Name:     Windows
 OS Version:  10.0.18362
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.508\

Host (useful for support):
  Version: 2.1.12
  Commit:  ccea2e606d

.NET Core SDKs installed:
  2.1.202 [C:\Program Files\dotnet\sdk]
  2.1.508 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.0.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]

### Issue

I tried to execute one sample using MLNET-CLI
`mlnet auto-train --task regression --dataset taxi-fare-train.csv --test-dataset taxi-fare-test.csv --label-column-name fare_amount --max-exploration-time 1000`

returns a error:
```
Best quality(RSquared): 0,9440, Best Algorithm: LightGbmRegression, Last Algorithm: FastTreeTweedieRegression
00:01:37 Exception occured while exploring pipelines:
All instances skipped due to missing features.
Please see the log file for more info.
```

### Source code / logs
Log: 
[debug_log.txt](https://github.com/dotnet/machinelearning/files/3611149/debug_log.txt)

Datasets: 
[teste.zip](https://github.com/dotnet/machinelearning/files/3611150/teste.zip)

#Update
I've also tryied using single datasets:

commands used:
`mlnet auto-train --task regression --dataset taxi-fare-test.csv --label-column-name fare_amount --max-exploration-time 1000`

`mlnet auto-train --task regression --dataset taxi-fare-train.csv --label-column-name fare_amount --max-exploration-time 1000`

logs:

[train_dataset_debug_log.txt](https://github.com/dotnet/machinelearning/files/3611873/train_dataset_debug_log.txt)
[test_dataset_debug_log.txt](https://github.com/dotnet/machinelearning/files/3611874/test_dataset_debug_log.txt)

**Comment by eerhardt** [comment]
cc @Dmitry-A @srsaggam - thoughts?

**Comment by julianobd** [comment]
It also happens on kali linux. 

> mlnet auto-train --task regression --dataset taxi-fare-train.csv --test-dataset taxi-fare-test.csv --label-column-name fare_amount --max-exploration-time 1000
```
SDK do .NET Core (refletindo qualquer global.json):
 Version:   2.2.402
 Commit:    c7f2f96116

Ambiente de tempo de execução:
 OS Name:     kali
 OS Version:  2019.4
 OS Platform: Linux
 RID:         linux-x64
 Base Path:   /usr/share/dotnet/sdk/2.2.402/

Host (useful for support):
  Version: 2.2.7
  Commit:  b1e29ae826

.NET Core SDKs installed:
  2.2.402 [/usr/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.2.7 [/usr/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.2.7 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.2.7 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download

```

**Comment by srsaggam** [comment]
@justinormont @JakeRadMSFT

**Comment by srsaggam** [comment]
https://github.com/dotnet/machinelearning/blob/d0b3f86b1d048612f2b255e52822ba37ca390b6a/src/Microsoft.ML.FastTree/FastTree.cs#L1906

Looks like this check may be failing .May be some one from ML.NET team  can tell what that check is about @codemzs @eerhardt  @justinormont

**Comment by srsaggam** [comment]
@CESARDELATORRE

**Comment by srsaggam** [comment]
@trbran

**Comment by trbran** [comment]
Seems like this is a result of using non-US regional settings. Simply switching to Country or Region = United States, and Regional format = English (United States) will solve it! I was running Norway/Norwegian and that consistently failed. Most likely date/time format and/or decimal separator related?

**Comment by julianobd** [comment]
@trbran yes, the problem was the regional settings, just tested here.. changed to English (United States) and the issue didnt happened.
